tests/python/ConfigTest.py

```diff
@@ -773,83 +775,287 @@ def test_virtual_display(self):
+        """
+
+        self.cfg.removeVirtualDisplayView('Raw')
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 1)
+        view_name = views[0]
+        self.assertEqual(view_name, 'Film')
+
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 1)
+        view_name = views[0]
+        self.assertEqual(view_name, 'sview2')
+
+        # Test remove a shared view from the Virtual Display.
+
+        self.cfg.removeVirtualDisplayView('sview2')
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 1)
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+        # Extra serialize & deserialize validation.
+        cfg = OCIO.Config().CreateFromStream(self.cfg.serialize())
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 1)
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+        self.cfg.addVirtualDisplaySharedView('sview2')
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 1)
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 1)
+
+    def test_remove_virtual_display(self):
+        """
+        Test remove the Virtual Display.
+        """
+
+        self.cfg.clearVirtualDisplay()
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 0)
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+        # Extra serialize & deserialize validation.
+        cfg = OCIO.Config().CreateFromStream(self.cfg.serialize())
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 0)
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+    def test_virtual_display_with_active_displays(self):
+        """
+        Test the virtual display instantiation when active displays
+        and views are defined.
+        """
+
+        self.cfg_active_display.validate()
+        displays = self.cfg_active_display.getDisplays()
+        self.assertEqual(displays.__len__(), 1)
+        views = self.cfg_active_display.getViews('sRGB')
+        self.assertEqual(views.__len__(), 1)
+
+    def test_virtual_display_v1(self):
+        """
+        Test that the virtual display is only supported by v2 or higher.
+        """
+
+        with self.assertRaises(OCIO.Exception):
+            cfg = OCIO.Config().CreateFromStream(SIMPLE_CONFIG_VIRTUAL_DISPLAY_V1)
+
+        cfg = OCIO.Config().CreateRaw()
```

Comment:
This is also a static method. This applies to all `OCIO.Config.Create*` methods used in this module.